doc/src/learn/tokenomics/gas-budgets.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Sui Gas Budgets
+---
+
+Gas Budgets are important because they guarantee that an user never pays more transaction fees than what they budget for. Sui’s gas mechanism is such that users pay upfront for both the current computational cost of executing a transaction and the long-term cost of storing the data associated with a transaction’s objects. Thus, Gas Budgets need to cover both sets of fees.
```

Comment:
```suggestion
A Sui transaction must pay for both the computational cost of execution and the long-term cost of storing the objects a transaction creates or mutates.
```

Reply:
Since this is so much shorter, I would suggest moving line 7 into this paragraph.